demo/components/presentational/SellOrder.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { shallow } from "enzyme";
+import SellOrder from "./SellOrder";
+
+describe("SellOrder", () => {
+  it("renders the component", async () => {
+    const sellOrder = {
+      id: -1,
+      priceMana: 0,
+      priceUsd: 0,
```

Comment:
Maybe `priceUSD` since USD is already an acronym

Reply:
Done.

Reply:
Thanks! Resolving.